src/main/riddl/Order/Order.riddl

```diff
@@ -4,16 +4,16 @@ context OrderContext is {
   type LineItem is {
     product: SKU,
     quantity: Number,
-    unitPrice: Number,
     lineTotal: Number
   }
 
   type OrderInfo is {
+    order_id: OrderId,
     lineItems: LineItem+,
-    specialInstructions: String,
```

Comment:
We probably don't need specialInstructions for the KalixStudy. However, I think we need it for future iterations. So rather than delete it, what if we made it optional? 
`specialInstructions: String?`

Reply:
sounds good!

Reply:
@miguelTaningcoYoppworks - please revert

Reply:
Reverted 👍